celesta-core/src/main/java/ru/curs/celesta/dbutils/BasicCursor.java

```diff
@@ -329,6 +330,15 @@ static BasicCursor create(DataGrainElement element, CallContext callContext, Set
         return (Class<? extends BasicCursor>) Class.forName(cursorClassName, true, Thread.currentThread().getContextClassLoader());
     }
 
+    protected static <T> ColumnRef<T> createColumnReference(final String columnName) {
```

Comment:
Is ``ColumnRef`` naming OK?

Reply:
as  I wrote you in Telegram we might rethink the thing )